tests/src/cpp_source/ut_test_gtest.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2023 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+#include <ut_kvp_profile.h>
+#include <ut.h>
+
+#define KVP_VALID_TEST_NO_FILE "assets/this_does_not_exist.yaml"
+#define KVP_VALID_TEST_YAML_FILE "assets/test_kvp.yaml"
+
+// Example of a derived test fixture class
+// Test fixture class derived from BaseTestFixture
+class UTKVPProfileTestL1 : public UTTestFixture {
+protected:
+    void SetUp() override {
+        UTTestFixture::SetUp();  // Call base class SetUp
```

Comment:
::SetUp()

Reply:
tried the above but compilation failed. Needs to be called with base class prefix like -> UTTestFixture::::SetUp()